resources/views/partials/content-single-lc_resource.blade.php

```diff
@@ -47,7 +50,7 @@
   </div>
   @endif
   <div class="resource__cta">
-    <p><a rel="external" class="cta" href="{{ Single::getPermanentLink() }}">{{ __('Visit full resource', 'coop-library') }} @svg('external', 'icon--external', ['focusable' => 'false', 'aria-hidden' => 'true'])</a></p>
+    <p><a rel="external" class="cta" href="{{ Single::getPermanentLink() }}">@if(Single::requiresSubscription()){{ __('Visit full resource (subscription required)', 'coop-library') }}@else{{ __('Visit full resource', 'coop-library') }}@endif @svg('external', 'icon--external', ['focusable' => 'false', 'aria-hidden' => 'true'])</a></p>
```

Comment:
Suggest adding a screen reader accessible label "Subscription required to access the resource".